cli/pcluster/numbers_to_list.py

```diff
@@ -0,0 +1,36 @@
+import collections
+import copy
+
+
+class NumberedList:
+    """
+    This class adds number to a list of string and associate each one with one element of the list. It can be used
+    to help a user decide an option faster.
+    """
+
+    def __init__(self, items):
+        """
+        Initialize the object with the list of string in which you want to add number.
+        :param items: the list of string in which number will be added
+        """
+        self.numbered_items = collections.OrderedDict()
+        for i in range(1, len(items) + 1):
+            self.numbered_items[i] = items[i - 1]
+
+    def get(self):
+        """
+        Return an ordered dict of the items with the numbers added.
+        :return: the ordered dict
+        """
+        return copy.copy(self.numbered_items)
```

Comment:
self.numbered_items.copy()

Reply:
Also what is the point of this copy? If you want to have an immutable list be aware of the difference between shallow and deep copy: https://docs.python.org/3.7/library/copy.html